README.md

```diff
@@ -1,24 +1,79 @@
-# README
+<div align="center">
+  <img src="https://user-images.githubusercontent.com/9601737/134082731-213dc876-abe9-4901-b314-9c1640abdf8d.png" alt="WNB.rb logo" height="200px" />
+  <h1>WNB.rb</h1>
+</div>
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+<p align="center">
+  <a href="https://twitter.com/wnb-rb">Follow us on Twitter</a> •
+  <a href="https://tinyurl.com/join-wnb-rb">Join Our Community</a>
+</p>
 
-Things you may want to cover:
+WNB.rb is a virtual community for women and non-binary Rubyists. This is the repo for WNB.rb's website, built by and for our community.
 
-- Ruby version
+## Code of Conduct
 
-- System dependencies
+All contributors to this repository must follow the [WNB.rb code of conduct](https://tinyurl.com/wnb-rb-coc). Those who violate the code of conduct will be reported to GitHub support and banned from all WNB.rb platforms, including GitHub and Slack.
 
-- Configuration
+## Getting Started
 
-- Database creation
+Before you try to set up this app, make sure you have the following installed on your machine:
+- Ruby (3.0.2)
+- Node (16.3.0)
+- Postgres (13.3)
+- Yarn
 
-- Database initialization
+### 1. Fork and Clone this repository
+If you're unfamiliar with this process, use the [GitHub docs](https://docs.github.com/en/get-started/quickstart/fork-a-repo) to get started!
 
-- How to run the test suite
+### 2. Install dependencies
+- To install Ruby dependencies, run: `bundle install`
+- To install JavaScript dependencies, run: `yarn install`
 
-- Services (job queues, cache servers, search engines, etc.)
+### 3. Set up the database
+- First, make sure Postgres is running on your machine
+- Then, run the following: `rails db:setup`
 
-- Deployment instructions
+### 4. Run the tests
+- To run the backend tests: `rspec`
+- To run the frontend tests: `yarn jest app/javascript`
 
-- ...
+### 5. Start the server
+To start both the frontend and backend servers at the same time, run:
+
+```
+foreman start
+```
+
+Alternatively, you may run the frontend and backend servers separately with these two commands:
+
+```
+rails s
+bin/webpack-dev-server
+```
+
+This may be helpful for debugging, but generally should not be necessary.
+
+## Contributing
+To contribute code to this repository, do the following:
+
+### 1. Perform all steps in the [Getting Started](#getting-started) section of this document
+
+### 2. Assign yourself an issue in our [Issues](https://github.com/wnbrb/wnb-rb-site/issues/new) tab
+Check out the `good first issue` tag if you're just getting started! Feel free to comment on an issue that interests you, and one of our maintainers can help you get started.
```

Comment:
Can we link [the "good first issue"](https://github.com/wnbrb/wnb-rb-site/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22) page here?